package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "git+https://github.com/discord/discord-api-docs.git"
   },
-  "author": "Discord Krew",
+  "author": "Discord Crew",
```

Comment:
I think this was misspelled by intent but correct me if I am wrong.
Also, a similar change was declined some time ago: https://github.com/discord/discord-api-docs/pull/1875.

Reply:
Oh ok